bin/style-dictionary.js

```diff
@@ -120,27 +142,33 @@ function getSD(configPath, options) {
   return new StyleDictionary(configPath, { verbosity, warnings });
 }
 
+/**
+ * @param {BuildOptions} [options]
+ */
 async function styleDictionaryBuild(options) {
   options = options || {};
   const configPath = getConfigPath(options);
   const sd = getSD(configPath, options);
 
   if (options.platform && options.platform.length > 0) {
-    return Promise.all(options.platforms.map((platform) => sd.buildPlatform(platform)));
+    Promise.all(options.platform.map((platform) => sd.buildPlatform(platform)));
```

Comment:
Doesn't the promise returned by `Promise.all` here also need to be `await`ed? (same for `styleDictionaryClean`)

Reply:
Yeah I don't think it matters coz this function is called by the CLI and then nothing else happens, the node process will end as soon as the stack is empty, but we should put an `await` just so it's less confusing to read